.github/workflows/build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: adopt
-          java-version: 17
+          java-version: 21
```

Comment:
can you do changes required for compile time as well.

Reply:
Updated the compile time jdk version to 21